xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -132,6 +156,35 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     DocumentBuilder domBuilder = factory.newDocumentBuilder();
     domBuilder.setErrorHandler(new FatalErrorHandler());
     Element dom = domBuilder.parse(xml).getDocumentElement();
+
```

Comment:
Hu? No that's not how Spring work(ed?)... Confused. 
But maybe I recall things wrong: XML snippets in there are only ones to be parsed by ehcache afaik. iirc Spring lets you configure `CacheManager` using their XML, no? Or it does so bootstrapping it pointing to our XML, but no substitution of theirs happens in ehcache.xml (and neither does it the other way around). 

Now I'm willing to be totally off and misremembering _everything_. If that's not the case though, I still think that consistency should rule and that, until proven otherwise, substitution should happen regardless of the schema of the element: i.e. if it's in the "ehcache.xml" it behaves consistently.